chat-plugins/info.js

```diff
@@ -1065,7 +1065,13 @@ var commands = exports.commands = {
 	weakness: function (target, room, user) {
 		if (!target) return this.parse('/help weakness');
 		if (!this.canBroadcast()) return;
-		var targets = target.split(/[ ,\/]/);
```

Comment:
`/\w*[,\/ ]?\w*/` doesn't work, probably because of how capturing groups work making the entire command the split pattern, but I used `/ ?[,\/ ] ?/` which is the same general effect.